packages/account-api/package.json

```diff
@@ -46,6 +46,7 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
+    "@metamask/base-controller": "^8.0.1",
```

Comment:
Required for the `Messenger`, but we will change that to `@metamask/messenger` once available.